src/worker/core/phase/newPhaseBeforeDraft.ts

```diff
@@ -50,6 +50,46 @@ const newPhaseBeforeDraft = async (
 			await idb.cache.players.put(p);
 		}
 	}
+	const challengeLoseBestPlayer = g.get("challengeLoseBestPlayer");
+	if (challengeLoseBestPlayer) {
+		const tids = g.get("userTids");
+		for (let i = 0; i < tids.length; i++) {
+			const tid = tids[i];
+			const [playersAll] = await Promise.all([
+				idb.cache.players.indexGetAll("playersByTid", tid),
+			]);
+			let pid = 0;
+			let bestOv = 0;
+			let bestPlayer: any = undefined;
+			//Find best player
+			playersAll.forEach((p: Player) => {
+				if (p.ratings.slice(-1)[0]["ovr"] > bestOv) {
+					bestOv = p.ratings.slice(-1)[0]["ovr"];
+					pid = p.pid;
+					bestPlayer = p;
+				}
+			});
+			const p: Player = bestPlayer as Player;
+			if (p.real) {
+				//If he´s real make it a free agent
+				p.contract.exp = g.get("season");
+				p.tid = PLAYER.FREE_AGENT;
+				let index = 0;
+				for (index = 0; index < p.salaries.length; index++) {
+					const sal = p.salaries[index];
+					if (sal["season"] == p.contract.exp) {
+						break;
+					}
+				}
+				p.salaries = p.salaries.slice(0, index + 1);
+			} else {
+				//if he´s not real kill him
+				p.diedYear = g.get("season");
+				await player.retire(p, conditions);
```

Comment:
Would be more fun to do a "tragic death" like in killOne.ts. The `killOne` function would have to be modified to accept a player as a parameter, currently it just picks a random player.